bob/src/hw_metrics_collector.rs

```diff
@@ -157,30 +167,35 @@ impl HWMetricsCollector {
     // NOTE: HashMap contains only needed mount points of used disks, so it won't be really big,
     // but maybe it's more efficient to store disks (instead of mount_points) and update them one by one
 
-    fn space(disks: &HashMap<PathBuf, DiskName>) -> DiskSpaceMetrics {
-        let mut total = 0;
-        let mut used = 0;
-        let mut free = 0;
-
-        for mount_point in disks.keys() {
+    /// Maps [`DiskSpaceMetrics`] to the corresponding mount point.
+    ///
+    /// `f_fsid` field of [`statvfs`] is used to ensure that disks are unique.
```

Comment:
According to the code of the function, you return mount point as the key. Please, fix the comment